Add staking transaction doc. Fix typo in transactions doc. (#71)
onflow/flow: docs/content/token/staking2/stakers.mdx

## 1. Fetch node operator information

> To fetch the node operator information for an address's staking proxy, the token holder must use use script TH.04:

Reviewer: It'd be good to add a bit of context as to why this step is required. Something like this:

> After a node operator has agreed to run a node for you, they will create a `NodeInfo` object holding the IP address and public keys for your new node. This `NodeInfo` object will be stored in an account owned by the node operator.
> 
> The node operator should provide you with their account `address` and the `nodeID` for your new node. These parameters can be passed into the `get_node_info` script (`TH.04`) to confirm the node information.
>
> If the node information is correct, continue to step 2.

Author: Added. Thank you!

`stakingProxy/sh_get_node_info.cdc`

Script arguments:

| Argument     | Type    | Purpose |
|--------------|---------|---------|

> | **account**  | Address | Specifies the address to get the staking proxy's node info node info for. |

Reviewer: ```suggestion
| **account**  | Address | The address of the node operator's account. |
```

IMO it's more clear to just say this is the node operator's account.

Author: Wow yes. I've changed this in rob/docs/staking-as-token-holder .

Script returns:

The `StakingProxy.NodeInfo` struct containing the details for the node. If no NodeInfo is available, the script will fail.

## 2. Register node & grant staking access to the node operator

> To register the node as the token holder account's staker, the token holder must use  use transaction TH.05:

Reviewer: Can you make it so all of the transaction IDs (e.g. `TH.05`) link to the relevant section of `transactions.mdx`? I know you can't link directly to the line in the table, but linking to the whole table is enough I think

Author: I've done this in rob/docs/staking-as-token-holder .

Author: Done.
